src/pudl/metadata/resources/eia861.py

```diff
@@ -697,9 +707,10 @@
         "etl_group": "eia861",
     },
     "core_eia861__yearly_short_form": {
-        "description": (
-            """This is data extracted from Form EIA-861S, a shorter version of Form EIA-861.
-The data started being reported in 2012. However, the 2019 data is not available.
+        "description": {
+            "additional_summary_text": "data from the short form (EIA-861S).",
+            "usage_warnings": ["missing_years"],
+            "additional_details_text": """The data started being reported in 2012. However, the 2019 data is not available.
```

Comment:
Do we still need to describe the link between EIA-861S and EIA-861 or do users already know that?

```suggestion
            "additional_details_text": """Form EIA-861S is a shorter version of Form EIA-861 that started being reported in 2012. However, the 2019 data is not available.
```